scripts/gn/README.md

```diff
@@ -0,0 +1,44 @@
+### Ginan Python CLI Tools
+
+This is a collection of tools to help you prepare your own data to run with Ginan.
+
+## Getting started
+
+Ensure you have created a virtual environment in the `scripts` folder and installed the `requirements.txt`.
```

Comment:
We could include instructions on how to do this via venv for example (or point people to the relevant docs that explain how to set up a virtual env)

Reply:
Just added some instructions to the README